lib/accelerator.js

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Veres One Accelerator API.
+ *
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const _ = require('lodash');
+const async = require('async');
+const bedrock = require('bedrock');
+const config = bedrock.config;
+const didv1 = require('did-veres-one');
+const fs = require('fs');
+const forge = require('node-forge');
+const logger = require('./logger');
+
+require('bedrock-express');
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock-express.ready', (app, callback) =>
+  _setupAccelerator(callback));
+
+// try to read and process, or generate save and process
+function _readOrCreateJson({path, create, loaded, stored}, callback) {
```

Comment:
Seems like this is a utility function that might find a better home in a lower-level lib like `did-veres-one` if nowhere else.

Reply:
Yes, I'd rather this be somewhere else.  Can revisit.